docs/01-app/02-guides/tailwind-css.mdx

```diff
@@ -23,7 +23,7 @@ npm install -D tailwindcss @tailwindcss/postcss postcss
 Create a `postcss.config.mjs` file in the root of your project and add the `@tailwindcss/postcss` plugin to your PostCSS configuration:
 
 ```js filename="postcss.config.mjs" highlight={4}
-/** @type {import('tailwindcss').Config} */
+/** @type {import('postcss-load-config').Config} */
```

Comment:
What I don't love about this one is that, we never mention that you should install `postcss-load-config`, so people doing this would get `any` type. 

- I wonder if it makes sense to delete this line altogether, 
- or, add a small subsection further down, explaining that if you want to get type hints for the config, then you need to install this extra package, and add this JSDoc comment.

I am not sure how many people are going to be updating their `postcss.config.mjs` file, enough to need type hints and such, at the cost of yet another dependency.

Reply:
> I wonder if it makes sense to delete this line altogether

I think this makes more sense based on what you said after about adding an additional dependency. So, if you’d like I can simply remove that line.

LMK and I’ll update the PR accordingly. Thanks for the review!